.github/workflows/node.js.yml

```diff
@@ -26,5 +26,6 @@ jobs:
                   node-version: ${{ matrix.node-version }}
                   cache: "npm"
             - run: npm ci
+            - run: npm lint
```

Comment:
Sorry, I meant to comment here. I think this should be `npm run lint`

Reply:
Yup, yarn habits die hard evidently 😅